### PR TITLE
Launch CFP

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

File: docs/conf/eu/2017/cfp.rst [hunk 1]
```diff
@@ -3,5 +3,113 @@
 Call for proposals
 ==================
 
-We're still ironing out the call for proposals for our North America conference in Portland,
-stay tuned for updates to this page as we get ready to launch the CFP for Europe!
+Hello hello, fellow documentarians! It's that time of year again: We’re very
```
Comment (thatdocslady) [inline docs/conf/eu/2017/cfp.rst:6]: Just a side note, I think it's kinda funny how the best way to figure out whether Sam or I wrote copy is by the "Hello hello" vs "Greetings" intro. <3

File: docs/conf/eu/2017/cfp.rst [hunk 2]
```diff
@@ -3,5 +3,113 @@
...
+excited to announce that we are now accepting talk proposals for our next North
+American conference, coming up on **September 10-12, 2017, in Prague, Czech Republic**.
```
Comment (thatdocslady) [inline docs/conf/eu/2017/cfp.rst]: North American > European

File: docs/conf/eu/2017/cfp.rst [hunk 3]
```diff
@@ -3,5 +3,113 @@
...
+
+Every year, Write the Docs invites people from all across our community to come
+up on stage to share their insights and experience. Whether you’ve been writing
+documentation for decades or you’ve just started this year, we want to hear from
+you! Read on to learn more about the goals of the conference and what we look
+for in talk proposals.
+
+In the meantime, mark your calendars: **The deadline for submitting proposals is
+Midnight PST on May 31st, 2017.** We aim to let you know if your proposal is
```
Comment (thatdocslady) [inline docs/conf/eu/2017/cfp.rst]: PST > CET

File: docs/conf/eu/2017/cfp.rst [hunk 4]
```diff
@@ -3,5 +3,113 @@
...
+accepted by 15th June.
+
+Conference goals
```
Comment (thatdocslady) [inline docs/conf/eu/2017/cfp.rst]: I'm assuming this is the same copy for NA, but EUified. Will skim but I doubt we need to make major changes here ;)

File: docs/conf/eu/2017/cfp.rst [hunk 5]
```diff
@@ -3,5 +3,113 @@
...
+Questions?
+----------
+
+If you have any questions, please email us at conf@writethedocs.org and
+let us know.
+
+Form
+----
+
+You can also view `our CFP <https://docs.google.com/forms/d/e/1FAIpQLSdphOZ5udg0LtuP-HVyJ2oybynNA_1eB_QRvVWMv45o5l5RZA/viewform>`_ in its own page.
```
Comment (thatdocslady) [inline docs/conf/eu/2017/cfp.rst:112]: Comment re: form - I suggest that in addition to saying "one paragraph is too short" we should add "and novels are too long" (or something more polite ;). General recommendation would be 2-3 short paragraphs.

File: docs/conf/eu/2017/cfp.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/eu/2017/cfp.rst:6]: This one is just copy pasted, so might even be Eric?

File: docs/conf/eu/2017/cfp.rst (hunk 4 above)
Comment (plaindocs) [reply docs/conf/eu/2017/cfp.rst]: Yup

File: docs/conf/eu/2017/cfp.rst (hunk 5 above)
Comment (plaindocs) [reply docs/conf/eu/2017/cfp.rst:112]: HA, I hovered over that sentence.

Comment (plaindocs) [reply docs/conf/eu/2017/cfp.rst:112]: > Tell us about your talk. Submitting a single paragraph doesn't give us much to go on, but a wall of text does not really help either.

**Review by thatdocslady — APPROVED** [review]
LGTM

**Pull request merged**